### PR TITLE
Expression to get ARGB values of colours

### PR DESCRIPTION
**Pull request by sovdeeth**
### Description

Adds expression to get ARGB values of any color. 
Adds API to Color to get ARGB values.

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** #5492

### CODE REVIEW

**Review by Efnilite — APPROVED** [review]
lgtm

**Review by erenkarakal — APPROVED** [review]

**Review by cheeezburga — APPROVED** [review]

File: src/main/java/ch/njol/skript/expressions/ExprARGB.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+@Examples({
+	"broadcast red value of rgb(100, 0, 50) # sends '100'",
+	"set {_red} to red's red value + 10"
+})
+@Keywords({"ARGB", "RGB", "color", "colour"})
+@Since("INSERT VERSION")
+public class ExprARGB extends SimplePropertyExpression<Color, Integer> {
+
+	static {
+		register(ExprARGB.class, Integer.class, "(:alpha|:red|:green|:blue) [value]", "colors");
```
Comment (Pikachu920) [inline src/main/java/ch/njol/skript/expressions/ExprARGB.java:31]: i'm thinking having value optional might lead to some odd usage

Comment (sovdeeth) [reply src/main/java/ch/njol/skript/expressions/ExprARGB.java:31]: I suppose `red's red` isn't great lol
i'll change it

Comment (Pikachu920) [reply src/main/java/ch/njol/skript/expressions/ExprARGB.java]: what do you think of `component` instead of value? this is what JS uses

Comment (sovdeeth) [reply src/main/java/ch/njol/skript/expressions/ExprARGB.java]: yeah that could be an alternative

**Review by Pikachu920 — APPROVED** [review]

File: src/main/java/ch/njol/skript/util/Color.java [hunk 2]
```diff
@@ -18,30 +18,49 @@
...
 
+import ch.njol.yggdrasil.YggdrasilSerializable.YggdrasilExtendedSerializable;
 import org.bukkit.DyeColor;
 import org.jetbrains.annotations.Nullable;
 
-import ch.njol.yggdrasil.YggdrasilSerializable.YggdrasilExtendedSerializable;
-
 public interface Color extends YggdrasilExtendedSerializable {
-	
+
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/util/Color.java:26]:
```suggestion
```
extraneous

File: src/main/java/ch/njol/skript/expressions/ExprARGB.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+})
+@Keywords({"ARGB", "RGB", "color", "colour"})
+@Since("INSERT VERSION")
+public class ExprARGB extends SimplePropertyExpression<Color, Integer> {
+
+	static {
+		register(ExprARGB.class, Integer.class, "(:alpha|:red|:green|:blue) (value|component)", "colors");
+	}
+
+	RGB color;
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/expressions/ExprARGB.java:34]:
```suggestion
	private RGB color;
```

**Review by APickledWalrus — APPROVED** [review]

**Review by Moderocky — APPROVED** [review]

**Pull request merged**